<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings_window</class>
 <widget class="QDialog" name="settings_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Налаштування</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(17, 15, 27);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="14,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,6,1">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>90</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,15">
         <item>
          <widget class="QCheckBox" name="basketCB">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="clearBasketLbl">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Автоматична очистка кошику після&lt;br/&gt;закриття програми&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,15">
         <item>
          <widget class="QCheckBox" name="historyCB">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="clearHistoryLbl">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Автоматична очистка списку виконаних&lt;br/&gt;нотаток післяз закриття програми&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,15">
         <item>
          <widget class="QCheckBox" name="notificationCB">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="notificationLbl">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Нагадування про особливі нотатки&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Таймер нагадування</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox {
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;

    background-color: rgb(235, 235, 235);
    border-color: rgb(196, 196, 196);
    color: rgb(119, 120, 115);
    padding-right: 30px;
}

QComboBox::drop-down {
    border-style: outset;
    border-width: 2px;
    border-radius: 4px;
    
    border: none;
    background-color: rgb(196, 196, 196);
    width: 30px;
    subcontrol-origin: padding;
    subcontrol-position: top right;
}

QComboBox::down-arrow {
    image: url(:/icon/icons/down_arrow.png);
    width: 20px;
    height: 10px;
}

QComboBox QAbstractItemView {
    background-color: rgb(235, 235, 235);
    color: rgb(119, 120, 115);
    selection-background-color: rgb(196, 196, 196);
    selection-color: rgb(255, 255, 255);
    padding: 5px;
    outline: none;
}
</string>
           </property>
           <item>
            <property name="text">
             <string>Раз в 5 хв</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Раз в 10 хв</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Раз в 15 хв</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Раз в 30 хв</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Раз в 60 хв</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Мова застосунку</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox {
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;

    background-color: rgb(235, 235, 235);
    border-color: rgb(196, 196, 196);
    color: rgb(119, 120, 115);
    padding-right: 30px;
}

QComboBox::drop-down {
    border-style: outset;
    border-width: 2px;
    border-radius: 4px;
    
    border: none;
    background-color: rgb(196, 196, 196);
    width: 30px;
    subcontrol-origin: padding;
    subcontrol-position: top right;
}

QComboBox::down-arrow {
    image: url(:/icon/icons/down_arrow.png);
    width: 20px;
    height: 10px;
}

QComboBox QAbstractItemView {
    background-color: rgb(235, 235, 235);
    color: rgb(119, 120, 115);
    selection-background-color: rgb(196, 196, 196);
    selection-color: rgb(255, 255, 255);
    padding: 5px;
    outline: none;
}
</string>
           </property>
           <item>
            <property name="text">
             <string>Українська</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>English</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Всі зміни в налаштуваннях нагадувань&lt;br/&gt;починають діяти лише при наступному&lt;br/&gt;запуску програми!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeBut">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	font-size: 12pt;
	border-style: outset;
    border-width: 2px;
    border-radius: 10px;
	
	background-color: rgb(152, 133, 248);
	color: rgb(0, 0, 0);
	border-color: rgb(17, 15, 27);
}
QPushButton:hover
{
	background-color: rgb(121, 97, 245);
}
QPushButton:pressed
{
	background-color: rgb(97, 72, 226);
}</string>
       </property>
       <property name="text">
        <string>Закрити</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
